frontend/src/pages/Tab.vue

```diff
@@ -15,31 +15,13 @@ import { BDropdown, BDropdownDivider, BDropdownItem } from "bootstrap-vue-next";
 import { notify } from "@kyvg/vue3-notification";
 import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
 import { isLoggedIn } from "../auth-client.js";
-
-const alphaTab = await import("@coderline/alphatab");
```

Comment:
Yes, but please keep the unrelated code unchanged, it is easier for me to review.


I forgot the reason, but I wrote this internationally to fix something.